### PR TITLE
feat: add redirect to github issues/bugs page

### PR DESCRIPTION
**Pull request by a-wallen**
### Feature Preview
image

**Issue** https://discord.com/channels/903549834160635914/903553722804748309/1119517258939645982
---

#### PR Checklist

- [X] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [X] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by appflowy** [comment]
How about moving to here?

image

**Comment by a-wallen** [comment]
Done @appflowy 
image

File: frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart [hunk 1]
```diff
@@ -195,21 +201,31 @@ extension QuestionBubbleExtension on BubbleAction {
         return LocaleKeys.questionBubble_shortcuts.tr();
       case BubbleAction.markdown:
         return LocaleKeys.questionBubble_markdown.tr();
+      case BubbleAction.github:
```
Comment (annieappflowy) [inline frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart:204]: Can the text be something like "Report a bug"? Not sure if the length is ok

Comment (a-wallen) [reply frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart:204]: How about Issues?

Comment (annieappflowy) [reply frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart:204]: It's a bit techie to me to be honest. If your non-developer friends can understand, it should be good enough.

Comment (annieappflowy) [reply frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart:204]: Other options I can think of:
1. Request
2. Feedback
3. Question?

**Pull request closed without merging**